P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1302 @@
+      chapter <ptr target="#SA"/>.</p>
+      <p>Removing timestamps from the text body can help meet
+        requirements of text anonymisation. For instance, if the <gi>particDesc</gi> and the
+        <gi>timeline</gi> are stored in a separate file, the rest of the corpus can be distributed
+        without this separate file. Thus the recipient of the corpus may know in what order posts
+        were made (if the values of the <att>synch</att> are sequential), and will be able to
+        group posts made by the same user, but will not have exact timestamps or actual user names,
+        thus providing a significant degree of anonymisation.
+      <egXML xmlns="http://www.tei-c.org/ns/Examples" xml:id="e10" xml:lang="en" source="#BIB_WPTalkAstronomicalObject">
+        <post modality="written" xml:id="cmc_post08" indentLevel="1" who="#u006" synch="#t006">
+          <p>Those haven't happened. If they do, we can revisit the concern. </p>
+          <signed generatedBy="template">
+            [_DELETED-SIGNATURE_]
+            <date synch="#t007">[_DELETED-TIMESTAMP_]</date>
+          </signed>
+        </post>
+      </egXML>
+      As demonstrated above, the <att>synch</att> attribute can be
+      used on <gi>date</gi> or <gi>time</gi> (or indeed any other
+      element) rather than on the <gi>post</gi> itself.</p>
+    </div>
+  </div>
+  <div xml:id="CMCrecs">
+    <head>Recommendations for Encoding CMC Microstructure</head>
+    <div xml:id="CMCemos">
+      <head>Emojis and Emoticons</head>
+      <p>Emojis are iconic or symbolic, invariant graphic units which the users of social media
+        applications such as WhatsApp, Instagram, and X (Twitter) can select from a menu or
+          <soCalled>emoji keyboard</soCalled> and embed into their written posts. Examples are
+        &#x1F601;, &#x1F637;, &#x01F308;, &#x1F631;, and &#x1F648;. An emoji is encoded by one or
+        more Unicode characters which are intended to be mapped directly to a pictorial symbol.</p>
+      <p>Emoticons predate emojis and are created as combinations of ASCII punctuation and other
+        characters using the keyboard. Examples are <code>:-)</code>, <code>;-)</code>,
+          <code>:-(</code>, <code>:-x</code>, <code>\O/</code>, and <code>Oo</code>. They first
+        occurred on a Usenet newsgroup (<ref target="#BIB_smiley">Fahlman, 2021</ref>) and then
+        became frequent in chat communications during the mid-1980s. An emoticon typically consists
+        of several Unicode characters (from the ASCII subset) in a row, each of which has an
+        intended use other than as part of an emoticon.</p>
+      <p>Both emoticons and emojis may be simply transcribed as a sequence of characters. As with
+        any other characters, they may be entered as numeric character entities if this is more
+        convenient. (E.g., <mentioned>&#x2764;;</mentioned> might be transcribed as
```

Comment:
One semicolon too much here: <mentioned>&#x2764;;</mentioned>

Reply:
Fixed in 759c2adba.